roles/artemis/templates/application-prod.yml.j2

```diff
@@ -357,3 +357,36 @@ management:
     metrics:
       export:
         enabled: true
+
+###############################################################################
+# Helios
+###############################################################################
+{% set is_build_agent = (IS_BUILD_AGENT is defined) %}
```

Comment:
Why are you not just using `is_build_agent` directly? Also this would mark `is_build_agent=false` as true here.